src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const allLists = document.querySelectorAll('li');
```

Comment:
Consider using a more specific selector if you only want to target certain list items. Using 'li' will select all list items in the document, which might not be the intended behavior if there are multiple lists.